showcase/src/app/demo/slider/slider.component.html

```diff
@@ -0,0 +1,36 @@
+<example-viewer [extensions]="['HTML', 'TS']" 
+				filesPath="slider/simple-slider"
+				title="SHOWCASE.DEMO.SLIDER.SIMPLE_SLIDER">
+	<div>
+		<h2 translate>SHOWCASE.DEMO.SLIDER.SIMPLE_SLIDER</h2>
+		<stark-slider [values]="simpleSliderValues"
+					  sliderId="simple-slider"
+					  [sliderConfig]="simpleSliderConfig"
+					  (changed)="onValuesChangeSimpleSlider($event)">
+		</stark-slider>
+		<div>
+			<label>Value:</label>
+			<input type="number" [value]="simpleSliderValues[0]" (change)="onChangeSimpleSlider($event)">
```

Comment:
Just a suggestion but I think it will be more consistent if we use Angular Material input across our examples.
https://material.angular.io/components/form-field/overview

@christophercr @SuperITMan What do you think?

Reply:
Indeed 👍